base/CMakeLists.txt

```diff
@@ -132,5 +127,6 @@ therock_provide_artifact(base
     rocm-cmake
     rocm-core
     rocm-half
+    rocprofiler-register
```

Comment:
Successful build on CI: https://github.com/nod-ai/TheRock/actions/runs/13319369250/job/37200838845?pr=94#step:11:622. 200 new lines of warning spam though.